src/lib/games/connect-four/ConnectFourGame.ts

```diff
@@ -262,6 +277,9 @@ export class Board {
 		return null;
 	}
 
+	private static readonly emojis: readonly string[] = ['1⃣', '2⃣', '3⃣', '4⃣', '5⃣', '6⃣', '7⃣'];
```

Comment:
You could cast this as const instead

Reply:
![image](https://user-images.githubusercontent.com/4019718/108401879-2082e780-721d-11eb-9d0e-0f9c50a70a93.png)